mysql-backup-restore-s3/run.sh

```diff
@@ -77,13 +77,28 @@ echo "=> Creating restore script"
 rm -f /restore.sh
 cat <<EOF >> /restore.sh
```

Comment:
@rasheedamir , @ozairaurora 
why are we creating a separate file restore.sh inside run.sh everytime, and execute in from within the same script everytime?